Doc/library/string.rst

```diff
@@ -588,6 +588,13 @@ The available presentation types for :class:`float` and
    |         | as altered by the other format modifiers.                |
    +---------+----------------------------------------------------------+
 
+The result should be correctly rounded to a given precision ``p`` of digits
+after the decimal point.  For :class:`float`, used rounding mode match the one
+for :func:`round` builtin, i.e. it should round results midway between
+representable values toward the nearest value with an even (0) least
+significant bit.  For :class:`~decimal.Decimal`, rounding mode of the current
+:ref:`context <decimal-context>` is used.
```

Comment:
```suggestion
The result should be correctly rounded to a given precision ``p`` of digits
after the decimal point.  The rounding mode for :class:`float` matches that
of the :func:`round` builtin.  For :class:`~decimal.Decimal`, the rounding
mode of the current :ref:`context <decimal-context>` will be used.
```

No need to restate what is already specified for `round`. People could just click on the corresponding link. Now just to be sure, the rounding being used is a ROUND_HALF_TO_EVEN right?

Reply:
Ok, so it's not a ROUND_HALF_TO_EVEN. But I don't know how to call it. It's ROUND_HALF_TO_EVEN_NDIGITS instead but is there an official name for it?

Reply:
> Ok, so it's not a ROUND_HALF_TO_EVEN.

It's ``FE_TONEAREST`` of Annex F or ``ROUND_HALF_EVEN`` in the decimal module.